docs/pages/versions/unversioned/sdk/video.mdx

```diff
@@ -249,6 +249,24 @@ const styles = StyleSheet.create({
 
 </SnackInline>
 
+## Caching videos (experimental)
+
+> **Info** Caching was introduced during the final stages of the beta phase for `expo-video`, and it has not been thoroughly tested by the community yet.
+> If you encounter any issues, please report them at the [expo/expo](https://github.com/expo/expo) repository.
```

Comment:
Long time has passed since then, `expo-video` is no longer in beta. Can you rephrase it a bit?